jotty/docker-compose.yml

```
@@ -0,0 +1,15 @@
+version: "3.7"
+services:
+  app_proxy:
+    environment:
+      APP_HOST: jotty_web_1
+      APP_PORT: "3000"
+  web:
+    image: ghcr.io/fccview/jotty:1.10.1@sha256:dddca0bed6eefe8ecfeb2286ad76fe529777aac1877fb8adabe86ede6daa2667
```

Run as user 1000, otherwise the app failed to start for me.
```suggestion
    image: ghcr.io/fccview/jotty:1.10.1@sha256:dddca0bed6eefe8ecfeb2286ad76fe529777aac1877fb8adabe86ede6daa2667
    user: "1000:1000"
```